### ISSUE TITLE
About training

### ISSUE THREAD

**Issue by DIO157128** [awsm-research/VulRepair#3]
Hi, @MichaelFu1998-create
I tried to re-run your experiment. I edited the script of M1 so that I can load the weights I trained from scratch.
`checkpoint_prefix = f'checkpoint-best-loss/{args.model_name}'
        output_dir = os.path.join(args.output_dir, '{}'.format(checkpoint_prefix))
        model.load_state_dict(torch.load(output_dir, map_location=args.device))
        model.to(args.device)
        test_dataset = TextDataset(tokenizer, args, file_type='test')
        test(args, model, tokenizer, test_dataset, best_threshold=0.5)`
I also edited the part of the tokenizer because I want to see its effect.
`   tokenizer = RobertaTokenizer.from_pretrained("Salesforce/codet5-base")
    model = T5ForConditionalGeneration.from_pretrained("Salesforce/codet5-base")
    `
But the result is quite strange. The accuracy reaches 42.79% using the code above. But the accuracy was only 38% using the original code. In another word, the model performs better when special tags like "<SS2V_StartBug>" are not added.
Do you have any clues why?

**Comment by MichaelFu1998-create** [comment]
Hi @DIO157128,
I'm not sure about the reasons for your case, but this could be possible as DL models are black-box models.
We did not empirically study the effect of special tokens such as "<S2SV_StartBug>" in our research.
However, this could be an interesting aspect to study.
